.storybook/stories/components/TabsStory.js

```diff
@@ -32,6 +35,7 @@ const TabsChapter = {
 // Story setup ================================================================
 const Story = () =>
 storiesOf('Tabs', module)
+    .addDecorator(withKnobs)
```

Comment:
Normally I tried to separate "playgrounds" (with knobs) from overview stories. However, I think it's fine in this case. Thansk for adding it 👍